site/content/en/docs/Installation/_index.md

```diff
@@ -31,15 +31,30 @@ AKS, while also ensuring that all Cloud providers can support that version.
 
 The following container operating systems and architectures can be utilised with Agones:
 
-
 | OS        | Architecture | Support    |
-|-----------|--------------|------------|
+| --------- | ------------ | ---------- |
 | linux     | `amd64`      | **Stable** |
 | linux     | `arm64`      | Alpha      |
 | [windows] | `amd64`      | Alpha      |
 
 For all the platforms in Alpha, we would appreciate testing and bug reports on any issue found.
 
+## Agones and Kubernetes Supported Versions
+
+Each version of Agones supports a specific version of Kubernetes. When a new version of Agones supports a new version of Kubernetes, it is explicitly called out in the [Agones version release notes](https://agones.dev/site/blog/releases/).
```

Comment:
nit: Maybe s/Agones version release notes/release notes/ (I think that the first part is implied and doesn't need to be explicit).